dev/breeze/src/airflow_breeze/utils/selective_checks.py

```diff
@@ -306,7 +306,7 @@ def full_tests_needed(self) -> bool:
     def python_versions(self) -> list[str]:
         return (
             CURRENT_PYTHON_MAJOR_MINOR_VERSIONS
-            if self.full_tests_needed
+            if self._run_everything or self.full_tests_needed
```

Comment:
It seems that `_run_everything` already checks for `self.full_tests_needed`. Is is or condition necessary ?

Same question for `all_python_versions`

Reply:
This is to handle the case where "run_everything = True" but "full_tests_needed = False" :)